theme-dev/SimpleStrings.md

```diff
@@ -0,0 +1,17 @@
+<h1 align="center">Simple Strings</h1>
+
+These are the simple strings, use them if you dont want to change multiple strings with the same color. Simple Strings basically group up multiple strings into one simple string
```

Comment:
maybe add short explanation that they are provided by the plugin/module so people don't go wondering where these come from

Reply:
does this sound good?
These strings are provided by the plugin and module and are under the "simple_colors" in the new theme layout